hermes-console/static/partials/topic.html

```diff
@@ -136,10 +135,8 @@ <h3 class="panel-title">Message schema</h3>
 
     <div class="row" ng-show="config.messagePreviewEnabled">
         <div class="col-md-12">
-            <div class="panel panel-default">
-                <div style="position: relative">
-                    <span class="btn pull-right glyphicon {{showMessageSchema ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down' }}" ng-click="showPreview = !showPreview"></span>
-                </div>
+            <div class="panel panel-default" style="position: relative">
+                <span class="btn pull-right glyphicon {{showPreview ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down' }}" ng-click="showPreview = !showPreview"></span>
```

Comment:
changed `showMessageSchema` to `showPreview` - schema expand used to change orientation of  preview panel chevron